src/components/notebook-utils.js

```diff
@@ -215,17 +216,20 @@ export const NotebookDuplicator = ajaxCaller(class NotebookDuplicator extends Co
         disabled: errors || processing,
         tooltip: Utils.summarizeErrors(errors),
         onClick: async () => {
+          this.setState({ processing: true })
           try {
-            this.setState({ processing: true })
             await (destroyOld ?
               Ajax().Buckets.notebook(namespace, bucketName, printName).rename(newName) :
-              Ajax().Buckets.notebook(namespace, bucketName, printName).copy(newName, bucketName, !destroyOld))
+              Ajax().Buckets.notebook(namespace, bucketName, printName).copy(newName, bucketName, !destroyOld)
+            )
             onSuccess()
             if (fromLauncher) {
               Nav.goToPath('workspace-notebook-launch', {
                 namespace, name: wsName, notebookName: `${newName}.ipynb`
               })
             }
+            const eventType = destroyOld ? Events.notebookRename : Events.notebookClone
+            Ajax().Metrics.captureEvent(eventType, { oldName: printName, newName })
```

Comment:
In order for these metrics to be valuable, we need to store workspace information, otherwise we won't be able to tell where these notebooks live. For rename, it's always within the same workspace, so there will just be one, but for copy, it can be between workspaces, so we'll want to store both source and destination.